### PR TITLE
Publish v6 boolean modules

### PR DESCRIPTION
**Pull request by DenisCarriere**
## Published v6 boolean modules

Ref: https://github.com/Turfjs/turf/issues/1265#issuecomment-367412034

@pwilczynski All of the `boolean` modules should now be published under `v6.x`, try them out and see if there's anything that fails.

### CODE REVIEW

File: packages/turf-boolean-parallel/package.json [hunk 1]
```diff
@@ -40,9 +40,9 @@
     "write-json-file": "*"
   },
   "dependencies": {
-    "@turf/clean-coords": "^5.1.5",
+    "@turf/clean-coords": "6.x",
     "@turf/helpers": "6.x",
-    "@turf/line-segment": "^5.1.5",
-    "@turf/rhumb-bearing": "^5.1.5"
+    "@turf/line-segment": "6.x",
+    "@turf/rhumb-bearing": "*"
```
Comment (DenisCarriere) [inline packages/turf-boolean-parallel/package.json]: Did not convert `rhumb-bearing` to Typescript yet

File: packages/turf-clean-coords/index.ts [hunk 2]
```diff
@@ -39,7 +43,7 @@ function cleanCoords(geojson, options) {
         });
         break;
     case 'MultiPolygon':
-        getCoords(geojson).forEach(function (polygons) {
+        getCoords(geojson).forEach(function (polygons: any) {
```
Comment (DenisCarriere) [inline packages/turf-clean-coords/index.ts]: `getCoords` is a bit wacky now, not outputting the correct type, `any` was added as a "hack" solution.

File: packages/turf-nearest-point-on-line/index.ts [hunk 3]
```diff
@@ -32,24 +44,18 @@ import { getCoords } from '@turf/invariant';
...
+): NearestPointOnLine {
+    var closestPt: any = point([Infinity, Infinity], {
         dist: Infinity
     });
 
     var length = 0.0;
-    flattenEach(lines, function (line) {
-        var coords = getCoords(line);
+    flattenEach(lines, function (line: any) {
+        var coords: any = getCoords(line);
```
Comment (DenisCarriere) [inline packages/turf-nearest-point-on-line/index.ts]: Another "hack" `getCoords` solution

**Comment by pwilczynski** [comment]
@DenisCarriere - just tried it out, was able to `yarn add @turf/boolean-within` and get 6.0.0, but doing the same with `yarn add @turf/boolean-overlap` chose 5.1.5 with the following output:

```
$ yarn add @turf/boolean-overlap
yarn add v1.3.2
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
success Saved lockfile.
success Saved 13 new dependencies.
├─ @turf/bearing@5.1.5
├─ @turf/boolean-overlap@5.1.5
├─ @turf/destination@5.1.5
├─ @turf/distance@5.1.5
├─ @turf/invariant@5.2.0
├─ @turf/line-intersect@5.1.5
├─ @turf/line-overlap@5.1.5
├─ @turf/line-segment@5.1.5
├─ @turf/meta@5.2.0
├─ @turf/nearest-point-on-line@5.1.5
├─ geojson-equality@0.1.6
├─ geojson-rbush@2.1.0
└─ rbush@2.0.2
✨  Done in 8.93s.
```

**Comment by pwilczynski** [comment]
@DenisCarriere would you mind letting me know when this merges?

**Comment by DenisCarriere** [comment]
@pwilczynski `boolean-overlap` is now published, must of forgotten it while I had to publish manually all these `boolean` modules.

**Comment by DenisCarriere** [comment]
Published new modules:
- [x] center
- [x] center-mean
- [ ] center-median (❌ npm test fail)
- [x] centroid
- [x] destination
- [x] distance

**Comment by pwilczynski** [comment]
@DenisCarriere -> awesome! Thanks so much!

**Comment by DenisCarriere** [comment]
Ref: https://github.com/Turfjs/turf/issues/1294

**Comment by DenisCarriere** [comment]
Finally a passing build 😓

**Pull request merged**